pkg/conf/natsconf.go

```diff
@@ -39,30 +39,54 @@ type TLSConfig struct {
 }
 
 type AuthorizationConfig struct {
-	Username string `json:"username,omitempty"`
-	Password string `json:"password,omitempty"`
-	Token    string `json:"token,omitempty"`
-	Timeout  int    `json:"timeout,omitempty"`
+	Username           string       `json:"username,omitempty"`
+	Password           string       `json:"password,omitempty"`
+	Token              string       `json:"token,omitempty"`
+	Timeout            int          `json:"timeout,omitempty"`
+	Users              []*User      `json:"users,omitempty"`
+	DefaultPermissions *Permissions `json:"default_permissions,omitempty"`
+}
+
+type User struct {
+	User        string       `json:"username,omitempty"`
+	Password    string       `json:"password,omitempty"`
+	Permissions *Permissions `json:"permissions,omitempty"`
+}
+
+// Permissions are the allowed subjects on a per
+// publish or subscribe basis.
+type Permissions struct {
+	Publish   []string `json:"publish,omitempty"`
+	Subscribe []string `json:"subscribe,omitempty"`
 }
 
 var (
 	ErrInvalidConfig = errors.New("natsconf: cannot produce valid config")
 )
 
 func Marshal(conf *ServerConfig) ([]byte, error) {
-	js, err := json.MarshalIndent(conf, "", "  ")
+	buf := &bytes.Buffer{}
+	encoder := json.NewEncoder(buf)
+	encoder.SetEscapeHTML(false)
+	err := encoder.Encode(conf)
```

Comment:
since `gnatsd` v1.1.0 is out, could wait for the Docker image to be available and remove all of this and just used JSON here (filed issue to remove: https://github.com/nats-io/nats-operator/issues/39)